src/Native/Unix/configure.cmake

```diff
@@ -362,6 +362,34 @@ check_prototype_definition(
 
 check_cxx_source_compiles(
     "
+    #include <stdlib.h>
+
+    int main()
+    {
+        char* path = strdup(\"abc\");
+        int ret = mkstemps(path, 3);
```

Comment:
Without the strdup, there would be a compiler warning-turned-error about casting a const char* to a char*